gridstatus/spp.py

```diff
@@ -1157,7 +1157,7 @@ def _get_marketplace_session(self) -> dict:
         """
         Returns a session object for the Marketplace API
         """
-        html = requests.get(MARKETPLACE_BASE_URL)
```

Comment:
There is now an issue with hitting this URL from Python. The response says "You need to enable JavaScript to run this app." and does not include the jsessionid or the csrf_token.